README.md

```diff
@@ -1,24 +1,70 @@
-# README
+# Recipe App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> The Recipe app keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe. Also, since sharing recipes is an important part of cooking the app should allow you to make them public so anyone can access them.
 
-Things you may want to cover:
+## Getting Started
```

Comment:
**Optional**
- It'd be awesome if you deployed your app and attach the live link in the README file

Reply:
This is actually an activity for Friday, so we won't do that yet.